docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -4,89 +4,92 @@
+Select the {anomaly-job} or the group of {anomaly-jobs} that is checked against 
+the rule. If you assign additional jobs to the group, the new jobs are 
+automatically checked the next time when the conditions are checked.
 
-You can select the result type of the {anomaly-job} that triggers the alert. 
-In particular, you can create alerts based on bucket, record, or influencer 
+You can select the result type of the {anomaly-job} that is checked against the 
+rule. In particular, you can create rules based on bucket, record, or influencer 
 results.
 
 [role="screenshot"]
 image::images/ml-anomaly-alert-severity.jpg["Selecting result type, severity, and test interval"]
 
-For each alert, you can configure the `anomaly_score` that triggers it. The 
-`anomaly_score` indicates the significance of a given anomaly compared to 
+For each rule, you can configure the `anomaly_score` that triggers the action. 
+The `anomaly_score` indicates the significance of a given anomaly compared to 
 previous anomalies. The default severity threshold is 75 which means every 
-anomaly with an `anomaly_score` of 75 or higher triggers the alert.
+anomaly with an `anomaly_score` of 75 or higher triggers the associated action.
 
-You can select whether you want the alert to include interim results. Interim 
-results are created by the {anomaly-job} before a bucket is finalized. These 
-results might disappear after the bucket is fully processed. Include 
-interim results if you want to be notified earlier about a potential anomaly 
-even if it might be a false positive. If you want to get notified 
-only about anomalies of fully processed buckets, do not include interim results.
+You can select whether you want to include interim results. Interim results are 
+created by the {anomaly-job} before a bucket is finalized. These results might 
+disappear after the bucket is fully processed. Include interim results if you 
+want to be notified earlier about a potential anomaly even if it might be a 
+false positive. If you want to get notified only about anomalies of fully 
+processed buckets, do not include interim results.
 
 You can also test the configured conditions against your existing data and check 
 the sample results by providing a valid interval for your data. The generated 
-preview contains the number of potentially created alert instances during the 
-relative time range you defined.
+preview contains the number of potentially created alerts during the relative 
+time range you defined.
 
 
 [[defining-actions]]
 == Defining actions
 
-As a next step, connect your alert to actions that use supported built-in 
-integrations. Actions are {kib} services or third-party integrations that run 
-when the alert conditions are met.
+As a next step, connect your rule to actions that use supported built-in 
+integrations by selecting a connector type. Connectors are {kib} services or 
+third-party integrations that perform an action when the alert conditions are 
```

Comment:
I think per [Connectors and actions](https://www.elastic.co/guide/en/kibana/master/action-types.html) it's rule conditions now not alert conditions:
```suggestion
third-party integrations that perform an action when the rule conditions are 
```